runtime/src/in_mem_accounts_index.rs

```diff
@@ -1122,6 +1202,13 @@ mod tests {
             );
             bucket.hold_range_in_memory(&ranges[0].clone(), false);
             assert!(bucket.cache_ranges_held.read().unwrap().is_empty());
+
+            // hold all in mem first
+            assert!(bucket.cache_ranges_held.read().unwrap().is_empty());
+            bucket.hold_range_in_memory(&all, true);
+            assert!(bucket.add_hold_range_in_memory_if_already_held(&range));
+            bucket.hold_range_in_memory(&range, false);
+            bucket.hold_range_in_memory(&all, false);
```

Comment:
I don't fully understand this whole test, but I did notice there wasn't an assert at the end. Is there a missing assert, or are these two lines cleanup?

Reply:
just getting back to steady state by deleting the ranges being held. We loop and keep going with other tests.